test/bench_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule BenchTest do
```

Comment:
Why do we need this test?

Reply:
to keep the test coverage up to 80% and to show it hat is possible to run the benchmark with a different spider